---
layout: page
title: Publications
# description: Sub-title
# background: '/img/publications.jpg'
---

<div class="paragraph">
    "*" indicates equal contribution
    <br />
    <em>Italics</em> indicate mentored lab member
    <br />
    <br />
    <strong>Preprints</strong>
    <br />
    <br />
    38. <em>Champer SE</em>*, <em>Kim IK</em>*, Clark AG, Messer PW, <strong>Champer J</strong>. 
    <a href="https://www.biorxiv.org/content/10.1101/2022.03.28.486009v1" target="_blank">Finding the strongest gene drive: Simulations reveal unexpected performance differences between <em>Anopheles</em> homing suppression drive candidates</a>. 
    <em>bioRxiv</em>, March 2022.
    <br />
    <br />
    37. <em>Liu Y</em>, <strong>Champer J</strong>. 
    <a href="https://www.biorxiv.org/content/10.1101/2021.10.12.464047v1" target="_blank">Modeling homing suppression gene drive in haplodiploid organisms</a>. 
    <em>bioRxiv</em>, October 2021.
    <br />
    <br />
    36. <em>Metzloff M</em>, <em>Yang E</em>, Dhole S, Clark AG, Messer PW, <strong>Champer J</strong>. 
    <a href="https://www.biorxiv.org/content/10.1101/2021.05.29.446308v1" target="_blank">Experimental demonstration of tethered gene drive systems for confined population modification or suppression</a>. 
    <em>bioRxiv</em>, May 2021.
    <br />
    <br />
    35. <em>Langmüller AM</em>*, <strong>Champer J</strong>*, <em>Lapinska S</em>, <em>Xie L</em>, <em>Metzloff M</em>, <em>Liu J</em>, <em>Xu Y</em>, Clark AG, Messer PW. 
    <a href="https://www.biorxiv.org/content/10.1101/2021.05.13.444039v1" target="_blank">Fitness effects of CRISPR endonucleases in <em>Drosophila melanogaster</em> populations</a>. 
    <em>bioRxiv</em>, May 2021.
    <br />
    <br />
    <br />
    <strong>Gene Drive Publications</strong>
    <br />
    <br /> 
    34. <em>Yang E</em>, <em>Metzloff M</em>, <em>Langmüller AM</em>, <em>Xu X</em>, Clark AG, Messer PW, <strong>Champer J</strong>. 
    <a href="https://academic.oup.com/g3journal/advance-article/doi/10.1093/g3journal/jkac081/6565321" target="_blank">A homing suppression gene drive with multiplexed gRNAs maintains high drive conversion efficiency and avoids functional resistance alleles</a>. 
    <em>G3</em>, 2022.
    <br />
    <br />   
    33. Wang GH, <em>Du J</em>, <em>Chu CY</em>, Madhav M, Hughes GL, <strong>Champer J</strong>.
    <a href="https://www.cell.com/trends/genetics/fulltext/S0168-9525(22)00040-3" target="_blank">Symbionts and gene drive: two strategies to combat vector-borne disease</a>.
    <em>Trends Genet</em>, 2022.
    <br />
    <br />  
    32. <em>Champer SE</em>, Oakes N, Sharma R, Garc&iacute;a-D&iacute;az P, <strong>Champer J</strong>, Messer PW.
    <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009660" target="_blank">Modeling CRISPR gene drives for suppression of invasive rodents</a>.
    <em>PLoS Comput Biol</em>, 17(12), e1009660, 2021.
    <br />
    <br />    
    31. Ferreira-Martins D*, <strong>Champer J</strong>*, McCauley DW, Zhang Z, Docker MF.
    <a href="https://www.sciencedirect.com/science/article/pii/S0380133021002379" target="_blank">Genetic control of invasive sea lamprey in the Great Lakes</a>.
    <em>J Great Lakes Res</em>, 47(S1), S764-S775, 2021.
    <br />
    <br />
    30. <strong>Champer J</strong>*, <em>Kim IK</em>*, <em>Champer SE</em>, Clark AG, Messer PW.
    <a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.15788" target="_blank">Suppression gene drive in continuous space can result in unstable persistence of both drive and wild-type alleles</a>.
    <em>Mol Ecol</em>, 30(4), 1086-1101, 2021.
    <br />
    <br />
    29. Long KC, Alphey L, Annas GJ, Bloss CS, Campbell KJ, <strong>Champer J</strong>, <em>et al</em>.  
    <a href="https://science.sciencemag.org/content/370/6523/1417" target="_blank">Core commitments for field trials of gene drive organisms</a>. 
    <em>Science</em>, 370(6523), 1417-1419, 2020.
    <br />   
    <br />  
    28. <strong>Champer J</strong>*, <em>Champer SE</em>*, <em>Kim IK</em>, Clark AG, Messer PW.
    <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/eva.13180" target="_blank">Design and analysis of CRISPR-based underdominance toxin-antidote gene drives</a>.
    <em>Evol Appl</em>, 14(4), 1052-1069, 2020.
    <br />
    <br />
    27. <strong>Champer J</strong>*, <em>Yang E</em>*, <em>Lee E</em>, <em>Liu J</em>, Clark AG, Messer PW.
    <a href="https://www.pnas.org/content/117/39/24377" target="_blank">A CRISPR homing gene drive targeting a haplolethal gene removes resistance alleles and successfully spreads through a cage population</a>.
    <em>Proc Natl Acad Sci U S</em>, 117(39), 24377-24383, 2020.
    <br />
    <br />
    26.<strong> Champer J</strong>, <em>Kim IK</em>, <em>Champer SE</em>, Clark AG, Messer PW.
    <a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-020-0761-2" target="_blank">Performance analysis of novel toxin-antidote CRISPR gene drive systems</a>.
    <em>BMC Biol</em>, 8(1), 27, 2020.
    <br />
    <br />
    25. <strong>Champer J</strong>, <em>Zhao J</em>, <em>Champer SE</em>, <em>Liu J</em>, Messer PW.
    <a href="https://pubs.acs.org/doi/10.1021/acssynbio.9b00452" target="_blank">Population dynamics of underdominance gene drive systems in continuous space</a>.
    <em>ACS Synth Biol</em>, 9(4), 779-792, 2020.
    <br />
    <br />
    24. <em>Champer SE</em>, <em>Liu C</em>, <em>Oh SY</em>, <em>Wen Z</em>, Clark AG, Messer PW, <strong>Champer J</strong>.
    <a href="https://advances.sciencemag.org/content/6/10/eaaz0525" target="_blank">Computational and experimental performance of CRISPR homing gene drive strategies with multiplexed gRNAs</a>.
    <em>Sci Adv</em>, 6(10), eaaz0525, 2020.
    <br />
    <br />
    23.<strong> Champer J</strong>, <em>Lee E</em>, <em>Yang E</em>, <em>Liu C</em>, Clark AG, Messer PW.
    <a href="https://www.nature.com/articles/s41467-020-14960-3" target="_blank">A toxin-antidote CRISPR gene drive system for regional population modification</a>.
    <em>Nat Commu</em>, 11(1), 1082, 2020.
    <br />
    <br />
    22. <strong>Champer J</strong>*, <em>Wen Z</em>*, <em>Luthra A</em>, <em>Reeves R</em>, <em>Chung J</em>, <em>Liu C</em>, <em>Lee YL</em>, <em>Liu J</em>, <em>Yang E</em>, Messer PW, Clark AG.
    <a href="http://www.genetics.org/content/early/2019/03/27/genetics.119.302037" target="_blank">CRISPR gene drive efficiency and resistance rate is highly heritable with no common genetic loci of large effect</a>.
    <em>Genetics</em>, 212(1), 334-341, 2019.
    <br />
    <br />
    21. <strong>Champer J</strong>, <em>Chung J</em>, <em>Lee YL</em>, <em>Liu C</em>, <em>Yang E</em>, <em>Wen Z</em>, Clark AG, Messer PW.
    <a href="https://elifesciences.org/articles/41439" target="_blank">Molecular safeguarding of CRISPR gene drive experiments</a>.
    <em>Elife</em>, 8, e41439, 2019.
    <br />
    <br />
    20. <em>Liu J</em>*, <strong>Champer J</strong>*, <em>Langmüller AM</em>, <em>Liu C</em>, <em>Chung J</em>, <em>Reeves R</em>, <em>Lee YL</em>, <em>Luthra L</em>, Clark AG, Messer PW.
    <a href="http://www.genetics.org/content/211/3/1005" target="_blank">Maximum likelihood estimation of fitness components in experimental evolution</a>.
    <em>Genetics</em>, 211(3), 1005-1017, 2019.
    <br />
    <br />
    19. <strong>Champer J</strong>*, <em>Liu J</em>*, <em>Oh SY</em>, <em>Reeves R</em>, <em>Luthra L</em>, <em>Oakes N</em>, Clark AG, Messer PW.
    <a href="http://www.pnas.org/content/early/2018/05/04/1720354115" target="_blank">Reducing resistance allele formation in CRISPR gene drive</a>.
    <em>Proc Natl Acad Sci U S A</em>, 115(21), 5522-5527, 2018.
    <br />
    <br />
    18. <strong>Champer J</strong>, <em>Reeves R</em>, <em>Oh SY</em>, <em>Liu C</em>, <em>Liu J</em>, Clark AG, Messer PW.
    <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006796" target="_blank">Novel CRISPR/Cas9 gene drive constructs reveal insights into mechanisms of resistance allele formation and drive efficiency in genetically diverse populations</a>.
    <em>PLoS Genetics</em>, 13(7), e1006796, 2017.
    <br />
    <br />
    17. <strong>Champer J</strong>, Buchman A, Akbari OS.
    <a href="https://www.nature.com/articles/nrg.2015.34" target="_blank">Cheating evolution: Engineering gene drives to manipulate the fate of wild populations</a>.
    <em>Nat Rev Genet</em>, 17, 146-159, 2016.
    <br />
    <br />
    <br />
    <strong>Other Publications</strong>
    <br />
    <br />
    16. <em>Yu Y</em>, Dunway S, <strong>Champer J</strong>, Kim J, Alikhan A.
    Changing our microbiome: Probiotics in dermatology.
    <em> Br J Dermatol</em>, 2019.
    <br />
    <br />
    15. <em>Champer M</em>, <em>Wong AM</em>, <strong>Champer J</strong>, Brito IL, Messer PW, Hou JY, Wright JD.
    The role of the vaginal microbiome in gynaecological cancer.
    <em>BJOG</em>, 125(3), 309-315, 2018.
    <br />
    <br />
    14. <em>Yu Y</em>*, <strong>Champer J</strong>*, Agak GW, Kao S, Modlin RL, Kim J.
    Different <em>Propionibacterium acnes</em> phylotypes induce distinct immune responses and express unique surface and secreted proteomes.
    <em> J Invest Dermatol</em>, 136(11), 2221-2228, 2016.
    <br />
    <br />
    13.<strong> Champer J</strong>, Ito JI, Clemons KV, Stevens DA, Kalkum M.
    Proteomic analysis of pathogenic fungi reveals highly expressed conserved cell wall proteins.
    <em>J. Fungi</em>, 2(1), 6, 2016.
    <br />
    <br />
    12. <em>Yu Y</em>, <strong>Champer J</strong>, Kim J.
    Analysis of the surface, secreted, and intracellular proteome of <em>Propionibacterium acnes</em>.
    <em>EuPA Open Proteom</em>, 9, 1-7, 2015.
    <br />
    <br />
    11. <em>Yu Y</em>, <strong>Champer J</strong>, Beynet DP, Kim J, Friedman AJ.
    The role of the cutaneous microbiome in skin cancer: Lessons learned from the gut.
    <em>J Drugs Dermatol</em>, 14(5), 461-465, 2015.
    <br />
    <br />
    10. <em>Yu Y</em>, <strong>Champer J</strong>, Garb&aacute;n H, Kim J.
    Typing of <em>Propionibacterium acnes</em>: A review of methods and comparative analysis.
    <em>Br J Dermatol</em>, 172(5), 1204-1209, 2015.
    <br />
    <br />
    9. Schmidt NW, Agak GW, Deshayes S, <em>Yu Y</em>, Blacker A, <strong>Champer J</strong>, Xian W, Kasko AM, Kim J, Wong GC.
    PenTobra: An aminoglycoside with robust antimicrobial &amp; membrane activity against <em>Propionibacterium acnes</em>.
    <em>J Invest Dermatol</em>, 135(6), 1581-1589, 2015.
    <br />
    <br />
    8. <em>Taylor EJM</em>, <em>Yu Y</em>, <strong>Champer J</strong>, Kim J.
    Resveratrol demonstrates antimicrobial effects against <em>Propionibacterium acnes</em>.
    <em>Dermatol Ther</em>, 4, 249-257, 2014.
    <br />
    <br />
    7. Chow A, Zhou W, Liu L, Fong MY, <strong>Champer J,</strong> Van Haute D, Chin AR, Ren X, Gugiu BG, Meng Z, Huang W, Ngo V, Kortylewski M, Wang SE.
    Macrophage immunomodulation by breast cancer-derived exosomes requires Toll-like receptor 2-mediated activation of NF-&kappa;B.
    <em>Sci Rep</em> 4, 5750, 2014.
    <br />
    <br />
    6. Lehrnbecher T, Kalkum M, <strong>Champer J</strong>, Tramsen L, Schmidt S, Klingebiel T.
    Immunotherapy in invasive fungal infection-focus on invasive aspergillosis.
    <em>Curr Pharm Des</em>, 19(20), 3689-3712, 2013.
    <br />
    <br />
    5. <strong>Champer J</strong>, <em>Patel J</em>, <em>Fernando N</em>, <em>Salehi E</em>, <em>Wong V</em>, Kim J.
    Chitosan against cutaneous pathogens.
    <em>AMB Express</em>, 3(1), 37, 2013.
    <br />
    <br />
    4. Friedman AJ, Phan J, Schairer DO, <strong>Champer J</strong>, Qin M, Pirouz A, Blecher-Paz K, Oren A, Liu PT, Modlin RL, Kim J.
    Antimicrobial and anti-inflammatory activity of chitosan-alginate nanoparticles: a targeted therapy for cutaneous pathogens.
    <em>J Invest Dermatol</em>, 133(5), 1231-1239, 2013.
    <br />
    <br />
    3. <strong>Champer J</strong>, Diaz-Arevalo D, <em>Champer M</em>, Hong TB, <em>Wong M</em>, <em>Shannahoff M</em>, Ito JI, Clemons KV, Stevens DA, Kalkum M.
    Protein targets for broad-spectrum mycosis vaccines: quantitative proteomic analysis of <em>Aspergillus</em> and <em>Coccidioides</em> and comparisons with other fungal pathogens.
    <em>Ann N Y Acad Sci</em>, 1273, 44-51, 2012.
    <br />
    <br />
    2. Chandra M, Zang S, Li H, Zimmerman L, <strong>Champer J</strong>, Chow A, Zhou W, Tsuyada A, Yu Y, Gao H, Ren X, Lin RJ, Wang SE.
    Nuclear translocation of type I TGF-&beta; receptor confers a novel function in RNA splicing.
    <em>Mol Cell Biol</em>, 32(12), 2183-2195, 2012.
    <br />
    <br />
    1. Bungau C, Camanzi B, <strong>Champer J</strong>, Chen Y, Cline DB, Luscher R, Lewin JD, Smith PF, Smith NJT, Wang H.
    Monte Carlo studies of combined shielding and veto techniques for neutron background reduction in underground dark matter experiments based on liquid noble gas targets.
    <em>Astroparticle Physics</em>, 23, 97-115, 2005.
    <br />
